src/components/ChattingScreen.js

```diff
@@ -18,14 +21,96 @@ export default function ChattingScreen() {
 		{ user: false, content: '나도 아니까  ' },
 		{ user: false, content: '그만 말해줘도 돼' },
 		{ user: true, content: 'ㅠㅠ' },
-	];
+	]);
+	function changeSender(e){
+		e.preventDefault();
+		setCurrentSender(!currentSender);
+	}
+	function changeNewMsg(e){
+		e.preventDefault();
+		setNewMsg({user: currentSender, content: e.target.value});
+	}
+	function updateMsglist(e){
+		e.preventDefault();
+		if(newMsg.content === ''){
+			alert('메시지를 입력해주세요');
+			return;
+		}
+		setCurrentMsglist([...currentMsglist, newMsg]);
+		setNewMsg({user: currentSender, content: ''});
+	}
+	let currentSenderName = currentSender ? USER_T_NAME : USER_F_NAME;
+	let currentSenderIMG = currentSender ? USER_T_IMG : USER_F_IMG;
+
+	useEffect(() => {
+		window.scrollBy(0, window.innerHeight);
+	}, [currentMsglist]);
 
 	return (
 		<Wrapper>
-			<Header></Header>
-			<MessageSender></MessageSender>
+			<Header sender = {currentSenderName} senderImg = {currentSenderIMG} changeSender = {changeSender} ></Header>
+			{currentMsglist.map((message) =>
+				<PrintOneMessage sender = {message.user} message = {message.content}/>
+			)}
+			<MessageSender message = {newMsg.content} changeNewMsg = {changeNewMsg} updateMsglist = {updateMsglist} ></MessageSender>
```

Comment:
(개인적인 생각입니다.)
message 대신 messageContent 어떤가요?? message를 인자로 내려보내면 마치 message 객체를 내리는 듯한 의미여서 messageContent로 텍스트를 내린다는 걸 확실하게 해주면 좋겠어요 ㅎㅎ

```suggestion
			<MessageSender messageContent = {newMsg.content} changeNewMsg = {changeNewMsg} updateMsglist = {updateMsglist} ></MessageSender>
```

Reply:
맞는 말이십니다!! 👍